### ISSUE TITLE
Tests for different platforms sometimes understand float assertions differently 

### ISSUE THREAD

**Issue by krzema12** [krzema12/PlotAssert#8]
See the code before this change: https://github.com/krzema12/fsynth/commit/9139cd3e86622c57c5cdf249a9e0745222219834

**Comment by krzema12** [comment]
Adding for the JVM beta milestone because trying to fix it later may influence PlotAssert's behaviou for JVM.

**Comment by krzema12** [comment]
For JVM (succeeds for JS):

```
For x=2.0: 0.20833334 is not between 0.125 and 0.20833333!
```

For JS (succeeds for JVM):

```
FailedConstraintException: For x=2: 0.20833333333333331 is not between 0.20833333333333334 and 0.29166666666666663!
```

Looks like prevision is different for JVM and JS. PlotAssert just evaluates the function and formally gets different values for x=2, depending on the platform. I don't think it's a bug in PlotAssert, but rather some peculiarity of using platform-specific number precision. For JS, it looks like double where for JVM it looks like float.

To sum up, I consider it not a bug of PlotAssert.
